labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -64,9 +68,13 @@ object CollectionExercise03 {
    * checkValuesIncrease(Seq(1,2,3)) == true
    * checkValuesIncrease(Seq(1,2,2)) == false
    */
-  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean =
-    error("fix me")
-
+  def checkValuesIncrease[T <% Ordered[T]](seq: Seq[T]): Boolean = {
+    seq match {
+      case Seq() => true
+      case Seq(_) => true
+      case _ => seq sliding 2 forall {case Seq(a, b) => a < b}
```

Comment:
This is way more interesting than my implementation.  I have no idea what it does (`sliding`?), but interesting.